python/cudf/cudf/tests/test_scalar.py

```diff
@@ -400,3 +400,13 @@ def test_datetime_scalar_from_string(data, dtype):
     expected = np.datetime64(datetime.datetime(2000, 1, 1)).astype(dtype)
 
     assert expected == slr.value
+
+
+def test_default_32bit_integer_scalar(default_32bit_integer):
```

Comment:
Same request as RangeIndex (and I'll ask you to copy it down to the below tests as well), add a docstring explaining what these tests do. The fixture is great, but realistically most people will start out confused.